includes/class-wp-statistics-helper.php

```diff
@@ -14,6 +14,21 @@
 
 class Helper
 {
+    /**
+     * List of add-ons.
+     *
+     * @var array
+     */
+    private static $addons = [
+        ['title' => 'Data Plus', 'key' => 'data-plus'],
+        ['title' => 'Real-time Stats', 'key' => 'realtime-stats'],
+        ['title' => 'Customization', 'key' => 'customization'],
+        ['title' => 'Advanced Reporting', 'key' => 'advanced-reporting'],
+        ['title' => 'Mini Chart', 'key' => 'mini-chart'],
+        ['title' => 'REST API', 'key' => 'rest-api'],
+        ['title' => 'Advanced Widgets', 'key' => 'widgets'],
+    ];
+
     /**
```

Comment:
Keep the add-ons hardcoded in the plugin, as we may introduce new add-on in future that will require updates to this list as well.

Additionally, the plugin already includes functionality to interact with add-ons. For example:
```
$pluginHandler = new \WP_Statistics\Service\Admin\LicenseManagement\Plugin\PluginHandler();
$pluginHandler->isPluginInstalled('data-plus');
```